src/redteam/plugins/base.ts

```diff
@@ -66,26 +68,38 @@ export abstract class PluginBase {
       });
 
       const finalTemplate = this.appendModifiers(renderedTemplate);
-
       const { output: generatedPrompts } = await this.provider.callApi(finalTemplate);
 
       invariant(typeof generatedPrompts === 'string', 'Expected generatedPrompts to be a string');
-      return generatedPrompts
-        .split('\n')
-        .filter((line: string) => line.includes('Prompt:'))
-        .map((line: string) => line.substring(line.indexOf('Prompt:') + 'Prompt:'.length).trim());
+      return this.parseGeneratedPrompts(generatedPrompts);
     };
     const allPrompts = await retryWithDeduplication(generatePrompts, n);
     const prompts = sampleArray(allPrompts, n);
     logger.debug(`${this.constructor.name} generating test cases from ${prompts.length} prompts`);
+    return this.promptsToTestCases(prompts);
+  }
+
+  protected promptsToTestCases(prompts: { prompt: string }[]): TestCase[] {
     return prompts.sort().map((prompt) => ({
       vars: {
-        [this.injectVar]: prompt,
+        [this.injectVar]: prompt.prompt,
       },
-      assert: this.getAssertions(prompt),
+      assert: this.getAssertions(prompt.prompt),
     }));
   }
 
+  protected parseGeneratedPrompts(generatedPrompts: string): { prompt: string }[] {
+    return generatedPrompts
+      .split('\n')
+      .filter((line: string) => line.includes('Prompt:'))
+      .map((line: string) => {
+        line = removeAsterisks(line);
```

Comment:
were you seeing this on a specific plugin or model?

Reply:
Whatever the default model is, I was seeing it on almost every single prompt response